pkg/osd/versions.go

```diff
@@ -89,6 +89,30 @@ func (u *OSD) LatestVersion(major, minor int64) (string, error) {
 	return VersionPrefix + latest.Original(), nil
 }
 
+// EnabledNoDefaultVersionList returns a sorted list of the enabled but not default versions currently offered by OSD.
+func (u *OSD) EnabledNoDefaultVersionList() ([]string, error) {
+	semverVersions, err := u.getSemverList(-1, -1, "")
```

Comment:
This list is not sorted -- it'll need to be sorted by semver in order to work properly. Sorry, just realized this today. Other than that and the other comment, I think this is good to go.

Reply:
Thanks for point out this..  It is my mistake that I forget doing sorting.

Reply:
@meowfaceman  You guys have achieved Sorting at the last of the function `getSemverList`: `sort.Sort(semver.Collection(versions))`

Reply:
Whoops, good catch. Sorry about that!